packages/volto/src/components/manage/Widgets/ObjectBrowserWidget.jsx

```diff
@@ -341,9 +351,14 @@ export class ObjectBrowserWidgetComponent extends Component {
             onChange(id, this.props.return === 'single' ? null : []);
           };
 
+    const props = {
+      ...this.props,
+      error: this.props.error.concat(this.state.errors),
+    };
+
     return (
       <FormFieldWrapper
-        {...this.props}
+        {...props}
```

Comment:
It's confusing to have a thing called `props` which is more than just the props. Can't you just pass `error` separately here?

Reply:
@davisagli check now it's better.

Thanks to the errors I've also found out that the OBW errors is messed up because how we use it as a HOC in some places :/ The errors shape switches back and forth from `{}` to `[]`... We didn't realise because we never validated what's inside. Now we are dealing with a little validation inside, and since we do not handle outside validation, let's simplify it and do not take that into consideration. We can do it if we enable external validation in this widget at some point.